tests/model/test_model.py

```diff
@@ -75,7 +78,11 @@ def test_init(self, model, initial_data, user_profile,
         assert model.unpinned_streams == []
         self.classify_unread_counts.assert_called_once_with(model)
         assert model.unread_counts == []
-        assert model.emoji_data == unicode_emojis
+        assert unicode_emojis.items() <= model.emoji_data.items()
+        assert custom_emojis.items() <= model.emoji_data.items()
```

Comment:
What are you trying to state by this?

Reply:
Makes sure that `model.emoji_data` contains both `unicode_emoji` and `custom_emoji`

Reply:
`set` logic seems appropriate for this, perhaps?